geo-location.html

```diff
@@ -67,13 +68,22 @@
         },
 
         /**
+         * If true, the element won't be active at all
+         */
+        idle: {
+          type: Boolean,
+          value: false,
+          observer: 'fetch'
```

Comment:
What does it mean to chance `idle` back to false after the location has been fetched? in `fetch()`, you clear the user's watch